src/Loader.php

```diff
@@ -1055,11 +1055,13 @@ public static function add_component_settings( $settings ) {
 			$setting_options = new \WC_REST_Setting_Options_V2_Controller();
 			foreach ( $preload_settings as $group ) {
 				$group_settings   = $setting_options->get_group_settings( $group );
-				$preload_settings = [];
-				foreach ( $group_settings as $option ) {
-					$preload_settings[ $option['id'] ] = $option['value'];
+				if ( ! is_wp_error( $group_settings ) ) {
```

Comment:
This will skip preloaded settings for any groups with errors which could have unwanted side effects downstream.